recipes/libastra/meta.yaml

```diff
@@ -0,0 +1,58 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+
+# Jinja variables help maintain the recipe as you'll update the version only here.
+# Using the name variable with the URL in line 14 is convenient
+# when copying and pasting from another recipe, but not really needed.
+{% set name = "libastra" %}
+{% set version = "1.9.9" %}
+{% set build = 0 %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/tomopy/astra-toolbox.git
+  git_tag: master
+
+build:
+  number: {{ build }}
+  string: "{{ "cuda" + cuda_compiler_version|string }}h{{ PKG_HASH }}_{{ build }}"
+  skip: true  # [osx or win or cuda_compiler_version == "None"]
+  merge_build_host: True  # [win]
+  script:
+    cmake
+    -G Ninja
+    -DCMAKE_BUILD_TYPE=Release
+    -DCMAKE_INSTALL_PREFIX:PATH=$PREFIX
+    -DCMAKE_INSTALL_LIBDIR=lib
```

Comment:
These are already part of `${CMAKE_ARGS}`
```suggestion
```

In general, it would be better to have this in a `build.sh` instead.

Reply:
> These are already part of ${CMAKE_ARGS}

That's good to know! I used them because they were in the [CMake section of the conda-forge knowledge base](https://conda-forge.org/docs/maintainer/knowledge_base.html#using-cmake). The contents of `${CMAKE_ARGS}` is undocumented(?), but it was in the[ CUDA section ](https://conda-forge.org/docs/maintainer/knowledge_base.html#cuda-builds).

Reply:
I was thinking that `build.sh` is only necessary when you have different scripts for Windows and Unix. In this case, I aim to use the same command to install on all platforms (if/when the CUDA toolchain comes to Windows on conda-forge).